app/src/test/java/uk/ac/ebi/atlas/search/metadata/HighchartsSunburstAdapterTest.java

```diff
@@ -0,0 +1,114 @@
+                                ImmutableList.of(species1.getName()),
+                                species1OrganismPart1ExperimentAccession)),
+                        ImmutableSet.of(ImmutablePair.of(
+                                ImmutableList.of(species1.getName()),
+                                species1OrganismPart2ExperimentAccession)),
+                        // Organism part 1
+                        ImmutableSet.of(ImmutablePair.of(
+                                ImmutableList.of(species1.getName(), species1OrganismPart1),
+                                species1OrganismPart1ExperimentAccession)),
+                        // Organism part 2
+                        ImmutableSet.of(ImmutablePair.of(
+                                ImmutableList.of(species1.getName(), species1OrganismPart2),
+                                species1OrganismPart2ExperimentAccession)),
+                        // Organism part 1, cell types
+                        ImmutableSet.of(ImmutablePair.of(
+                                ImmutableList.of(
+                                        species1.getName(), species1OrganismPart1, species1OrganismPart1CellType1),
+                                species1OrganismPart1ExperimentAccession)),
+                        ImmutableSet.of(ImmutablePair.of(
+                                ImmutableList.of(
+                                        species1.getName(), species1OrganismPart1, species1OrganismPart1CellType2),
+                                species1OrganismPart1ExperimentAccession)),
+                        ImmutableSet.of(ImmutablePair.of(
+                                ImmutableList.of(
+                                        species1.getName(), species1OrganismPart1, species1OrganismPart1CellType3),
+                                species1OrganismPart1ExperimentAccession)),
+                        // Organism part 2, cell types
+                        ImmutableSet.of(ImmutablePair.of(
+                                ImmutableList.of(
+                                        species1.getName(), species1OrganismPart2, species1OrganismPart2CellType1),
+                                species1OrganismPart2ExperimentAccession)));
+    }
+
+    @DisplayName("Cell type wheel should not have the same value for the ID and its parent")
+    @Test
+    void cellTypeWheelSunburstResultObjectShouldNotContainIdAndParentAttributeValuesSame() {
+        var result = subject.getCellTypeWheelSunburst(metadataSearchTerm,cellTypeWheelServiceMock
+                .search(metadataSearchTerm, species1.getName()));
+        assertThat(result).hasSize(2);
+        assertThat(result.asList().get(0).get("id")).isNotEqualTo(result.asList().get(0).get("parent"));
+        assertThat(result.asList().get(1).get("id")).isNotEqualTo(result.asList().get(1).get("parent"));
+    }
+
+    @Test
+    void ShouldThrowAssertionErrorIfWheelResultIdAndParentAttributeValuesAreSame() {
```

Comment:
Whenever a test fails, it throws an assertion error/exception, so this shouldn’t be part of what you’re testing. In other words, a test that is named:
```java
@Test
void throwsAssertionErrorIfTheSubjectWorksUnderSomeCondition() {
// ...
}
```

Should be renamed to:
```java
void subjectWorksUnderSomeCondition() {
// ...
}
```

One last observation: what’s the difference between this test and the previous one? What are you testing in each case? That should be your test name.